src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -99,9 +99,9 @@
     private final CaseDetailsConverter caseDetailsConverter;
     private final FeatureToggleService featureToggleService;
 
-    public StateFlow build() {
+    public StateFlow build(FlowState.Main state) {
         return StateFlowBuilder.<FlowState.Main>flow(FLOW_NAME)
-            .initial(DRAFT)
+            .initial(state)
```

Comment:
Could you please name the parameter as `initialState` instead of `state`

Reply:
yes, that make sense